langserver/completion.go

```diff
@@ -46,11 +47,21 @@ func (h *LangHandler) handleTextDocumentCompletion(ctx context.Context, conn jso
 		return nil, fmt.Errorf("invalid position: %s:%d:%d (%s)", filename, params.Position.Line, params.Position.Character, why)
 	}
 
-	ca, rangelen := gocode.AutoComplete(contents, filename, offset)
-	citems := make([]lsp.CompletionItem, len(ca))
-	for i, it := range ca {
+	ac, err := gocode.AutoComplete(&gocode.AutoCompleteRequest{
+		Filename: filename,
+		Data:     contents,
+		Cursor:   offset,
+		Builtin:  true,
```

Comment:
@keegancsmith I was missing this flag, after adding `Builtin: true` I was able to fix failing tests except whos with package autocomplete. There is one bug in gocode , if I place dot inside package path I get all builtins. In real world this is not a big deal and this can also be fixed with package autocomplete.

![image](https://user-images.githubusercontent.com/3028012/45281426-6da4f000-b4e0-11e8-8585-275c30c2313f.png)